src/fhirtypes/ElementDefinition.ts

```diff
@@ -1326,6 +1327,9 @@ export class ElementDefinition {
       const type = this.type[0].profile?.[0] ?? this.type[0].code;
       const def = resolve(type);
       if (def) {
+        if (!def.complete) {
+          logger.debug(`${def.name} is incomplete and may be missing required fields.`);
```

Comment:
I think now that since we are using `debug`, we want a command line option to set the log level to show `debug` messages. As it stands now, I don't think there is a way to see `debug` messages without manually changing app.ts to have `logger.level = 'debug'`.

Reply:
I'd suggest a slightly more specific log message:
```
logger.debug(`Warning: Circular relationship detected between ${this.name} and ${def.name}.  As a result, it is possible that the definition of ${this.name} may be based on incomplete components of ${def.name}.`); 
```